cmd/generate.go

```diff
@@ -166,7 +166,7 @@ var {{ .Camel }}Cmd = &cobra.Command{
 			return err
 		}
 
-		_, err = fmt.Fprint(os.Stdout, out)
+		_, err = fmt.Fprintf(os.Stdout, "%s\n", out)
```

Comment:
Can you please explain the use case of new line more. 
Thanks

Reply:
Hi, I changed `Fprint` to `Fprintf` is when using sttr in command line, the output have a '$' in the end of line.
eg:
![image](https://user-images.githubusercontent.com/26487341/200785608-b5b1e65e-0d8a-4f8c-8e45-10f9320b5c4c.png)

Reply:
and now it likes this
![image](https://user-images.githubusercontent.com/26487341/200785980-4a8718f7-a670-4fbf-a4d8-a3f3b6b3119b.png)